dockstore-webservice/src/main/resources/migrations.1.8.0.xml

```diff
@@ -80,4 +80,18 @@
         <addUniqueConstraint columnNames="imageid" constraintName="uk_imageid" tableName="entry_version_image"/>
         <addForeignKeyConstraint baseColumnNames="imageid" baseTableName="entry_version_image" constraintName="fk_entryVersionImage" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="image"/>
     </changeSet>
+
+    <changeSet author="esoth" id="notifications">
+        <createTable tableName="notification">
```

Comment:
Will need to co-ordinate with https://github.com/dockstore/dockstore/pull/3053/files#diff-75aaed69339ccc8cad3f7fa917e0cab2 @NatalieEO